packages/shared/src/components/layout/HeaderLogo.tsx

```diff
@@ -1,44 +1,22 @@
-import React, { ReactElement, useState } from 'react';
-import dynamic from 'next/dynamic';
-import { LoggedUser } from '../../lib/user';
+import React, { ReactElement } from 'react';
 import Logo, { LogoPosition } from '../Logo';
 import { useFeatureTheme } from '../../hooks/utils/useFeatureTheme';
 
-const Greeting = dynamic(
-  () => import(/* webpackChunkName: "greeting" */ '../Greeting'),
-);
-
 interface HeaderLogoProps {
-  user?: LoggedUser;
-  greeting?: boolean;
   onLogoClick?: (e: React.MouseEvent) => unknown;
   position?: LogoPosition;
 }
 
-function HeaderLogo({
-  user,
-  greeting,
-  onLogoClick,
-  position,
-}: HeaderLogoProps): ReactElement {
-  const [showGreeting, setShowGreeting] = useState(false);
+function HeaderLogo({ onLogoClick, position }: HeaderLogoProps): ReactElement {
```

Comment:
Maybe we can remove this, by moving `const featureTheme = useFeatureTheme();` into Logo?
And then just use Logo instead?

Reply:
Or maybe not, it might change the colour other places.

Reply:
Yeah it's used quite widely actually (Logo) that is.
Would prefer for now to keep it  maybe?

Reply:
Yeah, let's just keep it.